### PR TITLE
updates to software location

### PR DESCRIPTION
**Pull request by poraghal**

### CODE REVIEW

File: packages/gatsby-docs/src/pages/run/running_the_tool.mdx [hunk 1]
```diff
@@ -1,36 +1,67 @@
...
    
+   `docker load <./spm-ui-upgrade-helper.tar`
+
+3. Open Docker Desktop and check for the following images.
+
+4. Download the appropriate batch file or shell script for your environment:
+
+   Windows ™️
+
+   `curl -L https://raw.githubusercontent.com/IBM/spm-ui-upgrade-helper/main/spm-ui-upgrade-helper.bat -o spm-ui-upgrade-helper.bat`
```
Comment (luizas) [inline packages/gatsby-docs/src/pages/run/running_the_tool.mdx:30]: we don't need to download spm-ui-upgrade-helper.bat. or spm-ui-upgrade-helper.sh we need just docker-compose.yml

File: packages/gatsby-docs/src/pages/run/running_the_tool.mdx [hunk 2]
```diff
@@ -1,36 +1,67 @@
...
+
+   Linux ®️
+
    `curl -L https://raw.githubusercontent.com/IBM/spm-ui-upgrade-helper/main/spm-ui-upgrade-helper.sh -o spm-ui-upgrade-helper.sh`
 
-3. (Linux®️ only) Make the shell script executable with `chmod +x spm-ui-upgrade-helper.sh`
+5. (Linux®️ only) Make the shell script executable with `chmod +x spm-ui-upgrade-helper.sh`
```
Comment (luizas) [inline packages/gatsby-docs/src/pages/run/running_the_tool.mdx:36]: we don't need this step too

File: packages/gatsby-docs/src/pages/run/running_the_tool.mdx [hunk 3]
```diff
@@ -1,36 +1,67 @@
...
 
-4. Download the docker-compose.yml file:
+6. Download the docker-compose.yml file:
 
    Windows ™️
 
-   `curl -L https://raw.githubusercontent.com/IBM/spm-ui-upgrade-helper/main/docker-compose.yml -o docker-compose.yml`
+   `curl -L curl -L https://raw.githubusercontent.com/merative/spm-ui-upgrade-helper/main/docker-compose.yml -o docker-compose.yml -o docker-compose.yml`
```
Comment (luizas) [inline packages/gatsby-docs/src/pages/run/running_the_tool.mdx:42]: we need to have curl -L once in command

File: packages/gatsby-docs/src/pages/run/running_the_tool.mdx [hunk 4]
```diff
@@ -1,36 +1,67 @@
...
+
+  Linux ®️
+
+  ```bash
+   docker-compose up --no-build 
+  ```
+  In Docker Desktop, you can see the running containers.
 
-6. Set the required variables and run the batch file or shell script:
+1.  Set the required variables and run the batch file or shell script:
```
Comment (luizas) [inline packages/gatsby-docs/src/pages/run/running_the_tool.mdx:64]: we should set up variables before we will run docker-compose up --no-build

File: packages/gatsby-docs/src/pages/run/running_the_tool.mdx [hunk 5]
```diff
@@ -50,7 +81,7 @@ Complete the following steps to configure and run the tool and to review and imp
   Linux ®️
 
   ```bash
-  export INPUT_FOLDER=`<input_folder>
+  export INPUT_FOLDER=`<input_folder>`
   export OUTPUT_FOLDER=`<output_folder>`
   export VERSION=`<version>` 
   sh spm-ui-upgrade-helper.sh
```
Comment (luizas) [inline packages/gatsby-docs/src/pages/run/running_the_tool.mdx:87]: we are not running spm-ui-upgrade-helper.sh. /spm-ui-upgrade-helper.bat script

**Comment by poraghal** [comment]
@luizas Thanks Luiza, have addressed your comments, see latest version

**Review by luizas — APPROVED** [review]

**Pull request merged**